.tool-versions

```diff
@@ -0,0 +1,4 @@
+ruby 2.7.5
+yarn 1.22.18
+nodejs 12.22.12
+npm 6.14.16
```

Comment:
I dont think we need to specify npm in the tool-versions

Reply:
Ok, removed, thanks!